composer.json

```diff
@@ -211,6 +211,7 @@
             "rules/cakephp-to-symfony/tests/Rector/Class_/CakePHPModelToDoctrineEntityRector/Source"
         ],
         "files": [
+            "packages/better-php-doc-parser/tests/PhpDocInfo/PhpDocInfoPrinter/AbstractPhpDocInfoPrinterTest.php",
```

Comment:
Why this file? It should be auotloaded by PSR-4

Reply:
Because it is a function and functions are not autoloaded by PHP.

Some references about this problem (that took me many days of researches before understanding what was that didn't work 🤬):

- "This is useful if your package includes PHP functions that cannot be autoloaded by PHP." (https://getcomposer.org/doc/04-schema.md#files)
- https://github.com/phpstan/phpstan/issues/95#issuecomment-274257189
- https://github.com/Roave/BetterReflection#limitations
- https://github.com/thecodingmachine/safe/blob/master/composer.json#L13-L99